tests/integration/targets/docker_image/templates/ArgsDockerfile

```diff
@@ -0,0 +1,13 @@
+# Copyright (c) Ansible Project
```

Comment:
Do you want to replace `Ansible Project` with your own name?

Reply:
Yeah, why not. (Not sure whether this file actually meets the threshold of originality...)
```suggestion
# Copyright (c) 2022, Felix Fontein
```